app/src/views/Login/Login.vue

```diff
@@ -183,17 +190,17 @@ export default {
       const redirectPath = this.$route.query.redirect
 
       if (redirectPath === '/wallet/transfer') {
-        return WalletTransferLoader
+        return this.$vuetify.breakpoint.xsOnly ? WalletTransferLoaderMobile : WalletTransferLoader
```

Comment:
How are you tracking theme here to set dark mode loaders?

Reply:
Its always light mode here since this is during login, theme wasn't fetched yet

Reply:
I'm unable to demo this lionell. can you please help me see this loader?

Reply:
Please add a loader for /redirect as well

Reply:
> Please add a loader for /redirect as well

Done